docs/source/guides/performance.rst

```diff
@@ -84,6 +84,8 @@ When an entire dataset is not required to calculate the features for a given set
 
 An example of this approach can be seen in the `Predict Next Purchase demo notebook <https://github.com/featuretools/predict_next_purchase>`_. In this example, we partition data by customer and only load a fixed number of customers into memory at any given time. We implement this easily using `Dask <https://dask.pydata.org/>`_, which could also be used to scale the computation to a cluster of computers. A framework like `Spark <https://spark.apache.org/>`_ could be used similarly.
 
+An additional example of partitioning data to distribute on multiple cores or a cluster using Dask can be seen in the `Featuretools on Dask notebook <https://github.com/Featuretools/Automated-Manual-Comparison/blob/master/Loan%20Repayment/notebooks/Featuretools%20on%20Dask.ipynb>`_. Dask allows us to easily scale to multiple cores on a single computer or multiple machines on a cluster. 
```

Comment:
let's also mention there is a blog post about it and link to the TDS blog as well

Reply:
Link to the TDS blog or the Feature Labs engineering blog?

Reply:
actually, let's do Feature Labs engineering blog